Bindings/Java/OpenSimJNI/CMakeLists.txt

```diff
@@ -4,20 +4,22 @@ set(UKIT JAVAJNI)
 # For building C++ wrapper.
 find_package(JNI 1.6 REQUIRED)
 
+function(GenerateJavaWrapper
```

Comment:
Our convention for naming cmake macros/functions would give `OpenSimGenerateJavaWrapper`.